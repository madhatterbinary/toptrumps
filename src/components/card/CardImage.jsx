import React from 'react';
import PropTypes from 'prop-types';
import images from '../../images/index';

const CardImage = (props) => (
  <div className="card-image-container">
    <img src={images[props.image]} className="card-image" alt="" />
  </div>
);

CardImage.propTypes = {
  image: PropTypes.string,
}

export default CardImage;
