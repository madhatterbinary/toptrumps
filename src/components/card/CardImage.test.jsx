import React from 'react';
import CardImage from './CardImage'
import TestRenderer from 'react-test-renderer';

describe('Card', () => {
  it('renders', () => {
    const component = TestRenderer.create(<CardImage image="" />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
