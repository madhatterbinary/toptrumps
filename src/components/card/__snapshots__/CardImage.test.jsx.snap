// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders 1`] = `
<div
  className="card-image-container"
>
  <img
    alt=""
    className="card-image"
  />
</div>
`;
